sktm/jenkins.py

```diff
@@ -159,136 +154,139 @@ def __get_cfg_data_max(self, jobname, buildid, stepname,
             buildid:    Jenkins build ID.
 
         Return:
             The epoch timestamp string of the committer date.
         """
-        return self.__get_cfg_data_uniform(jobname, buildid, "skt.cmd_merge",
+        return self.__get_cfg_data_uniform(buildid, "skt.cmd_merge",
                                            "commitdate")
 
-    def get_base_hash(self, jobname, buildid):
+    def get_base_hash(self, buildid):
         """
-        Get base commit's hash of the specified completed build for the
-        specified project. Wait for the build to complete, if it hasn't yet.
+        Get base commit's hash of the specified completed build.
+        Wait for the build to complete, if it hasn't yet.
 
         Args:
-            jobname:    Jenkins project name.
             buildid:    Jenkins build ID.
 
         Return:
             The base commit's hash string.
         """
-        return self.__get_cfg_data_uniform(jobname, buildid, "skt.cmd_merge",
+        return self.__get_cfg_data_uniform(buildid, "skt.cmd_merge",
                                            "basehead")
 
-    # FIXME Clarify function name
-    def get_patchwork(self, jobname, buildid):
+    def get_patch_url_list(self, buildid):
         """
-        Get the list of Patchwork patch URLs for the specified completed build
-        for the specified project. Wait for the build to complete, if it
-        hasn't yet.
+        Get the list of Patchwork patch URLs for the specified completed
+        build. Wait for the build to complete, if it hasn't yet.
 
         Args:
-            jobname:    Jenkins project name.
             buildid:    Jenkins build ID.
 
         Return:
-            The list of Patchwork patch URLs.
+            The list of Patchwork patch URLs, in the order the patches should
+            be applied in.
         """
-        return self.__get_cfg_data_uniform(jobname, buildid, "skt.cmd_merge",
-                                           "pw")
+        return self.__get_cfg_data_uniform(buildid, "skt.cmd_merge", "pw")
 
-    def get_baseretcode(self, jobname, buildid):
+    def get_baseretcode(self, buildid):
         """
         Get the maximum (the worst) return code of a baseline test across all
         "run" steps for the specified completed build of the specified
         project. Wait for the build to complete, if it hasn't yet.
         """
-        return self.__get_cfg_data_max(jobname, buildid, "skt.cmd_run",
+        return self.__get_cfg_data_max(buildid, "skt.cmd_run",
                                        "baseretcode", 0)
 
-    def get_result_url(self, jobname, buildid):
-        return "%s/job/%s/%s" % (self.server.base_server_url(), jobname,
+    def get_result_url(self, buildid):
+        """
+        Get the URL of the web representation of the specified build.
+
+        Args:
+            buildid:    Jenkins build ID.
+
+        Return:
+            The URL of the build result.
```

Comment:
I can't comment on a specific commit, but the `Describe sktm.jenkins.get_result_url functions ` commit is only adding docstring to this function, while in the original commit it's also documenting `get_result` one (hence why there is an asterisk in the original commit message, which is also missing in this one).